File: src/main/java/se/ifmo/pepe/dto/UserResponseDTO.java
package se.ifmo.pepe.dto;

import java.util.List;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import se.ifmo.pepe.model.Role;

@Data
public class UserResponseDTO {

  @ApiModelProperty(position = 0)
  private Integer id;
  @ApiModelProperty(position = 1)
  private String username;
  @ApiModelProperty(position = 2)
  List<Role> roles;


}

File: src/main/java/se/ifmo/pepe/dto/UserResponseDTO.kt
package se.ifmo.pepe.dto

import io.swagger.annotations.ApiModelProperty
import se.ifmo.pepe.domain.Role

data class UserResponseDTO(
    @ApiModelProperty(position = 0)
    val id: Int,
    @ApiModelProperty(position = 1)
    val username: String,
    @ApiModelProperty(position = 2)
    val roles: List<Role>
)
